src/encoded/static/components/status.js

```diff
@@ -108,6 +108,7 @@ const defaultObjectStatuses = {
             'released',
             'archived',
             'revoked',
+            'preferred default',
```

Comment:
This section is only for file status, but “preferred default” can’t be a file status.

Reply:
Done.